api/src/application/user/user-views.js

```diff
@@ -0,0 +1,229 @@
+const querystring = require('querystring');
+const formatError = require('../../lib/format-error');
+const errorMessages = require('./user-error-messages');
+const get = require('lodash/get');
+const userFormData = require('./user-form-data');
+
+const getValidationMessages = function(error) {
```

Comment:
Could this be shared across other view files in the future? Might make sense to move it to a lib folder so it's more discoverable/doesn't get re-used

Reply:
That was moved out of the controller file. It _might_ get reused. I'd rather wait until it needs to be reused to pull it out.

I'm not worried about discoverability so much because theoretically the person who needs it in their views will be copying the pattern from this file and we'll abstract it at that point.